client/src/components/SignUpAd/SignUpAd.css

```diff
@@ -1,10 +1,9 @@
 .SignUpAd {
-  /* height: 47vh; */
   background-color: var(--yellow);
-  width: 100vw;
   text-align: center;
   font-family: var(--default-font);
-  letter-spacing: .1rem;
+  letter-spacing: 0.1rem;
+  padding: 10px;
 }
 
 .SignUpAd .inputs {
```

Comment:
You're overwriting styles in another component here. This will be really hard to debug in the future. Remember that we want our components to be atomic!

Reply:
Hey @ZakkMan not sure what component is being overriden - would you mind clarifying?

Reply:
@caioingber This one is good. The `input` element is part of the `Input` component, so if you want to change the `input` (the element), you should change `Input` (the component)